scripts/install-deps.sh

```diff
@@ -216,6 +219,7 @@ install_deps_arch()
         fmt \
         fontconfig \
         git \
+        gsl \
```

Comment:
Many thanks for also taking care of other platforms. - A safe exit would be to always embed a dependency, but that should be last resort.